src/runtime/d3d12_abi_patch_64.h

```diff
@@ -0,0 +1,68 @@
+#ifndef HALIDE_HALIDERUNTIMED3D12COMPUTE_ABIPATCH64_H
+#define HALIDE_HALIDERUNTIMED3D12COMPUTE_ABIPATCH64_H
+
+// D3D12 ABI patch trampolines
+// (refer to 'd3d12_abi_patch_64.ll')
+
```

Comment:
I think this file could use some explanatory comments: why are these trampolines required?  This will help for future documentation/preservation of the work you had to go through to reverse engineer the need for trampolining

Reply:
I added some comments to the `d3d12_abi_patch_64.h` and `d3d12_abi_patch_64.ll` files.